page_objects/page_ovirt_hostedengine.py

```diff
@@ -93,19 +89,25 @@ class OvirtHostedEnginePage(PageTest):
     STORAGE_GLUSTERFS = _STORAGE_TYPE % 'glusterfs'
 
     # FINISH STAGE
-    FINISH_DEVELOPMENT = "XPATH{}//button[@class='btn btn-primary wizard-pf-next']"
-    CLOSE_BUTTON = "XPATH{}//button[@class='btn btn-primary wizard-pf-next']"
+    FINISH_DEVELOPMENT = "XPATH{}//button[text()='Finish Development']"
+    CLOSE_BUTTON = "XPATH{}//button[text()='Close']"
 
     # CHECKPOINTS
     MAINTENANCE_HINT = "XPATH{}//div[text()='Local maintenance cannot be enabled with a single host']"
-    GLOBAL_HINT = "XPATH{}//span[contains(@class, 'pficon-warning-triangle-o')]"
+    GLOBAL_HINT = "XPATH{}//div[text()='The cluster is in global maintenance mode!']"
```

Comment:
[contains(text(), '')] is better than [text()=''], for the text may be changed a little bit sometimes. But if there are more than one similar text on the page, maybe have to use [text()=''].